docusaurus/docs-internals/editor-controls/unit-controls/slider.md

```diff
@@ -0,0 +1,61 @@
+# Slider
+
+Slider control allows users to select a numerical value by dragging a slider handle. Users can set minimum and maximum values, step intervals, and units of measurement. Additionally, the component supports both immediate and debounced updates to optimize performance and user experience.
```

Comment:
replace "Users" word, because in this context "Users" are understood as the end users, not the developers
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/12895e24-cce0-4d5c-9326-3354108db64d)

Reply:
replaced with `Developers`